common/web/lm-worker/src/main/model-compositor.ts

```diff
@@ -559,6 +568,53 @@ export default class ModelCompositor {
     return suggestions;
   }
 
+  private predictionAutoSelect(suggestionDistribution: CorrectionPredictionTuple[]) {
```

Comment:
Potential followup:

Note that this method does not actually reference anything in the class.  We could make it static... or even better, we could make it a standalone method.

Why this point is significant:  it's isolated in a manner great for unit testing.  No need to mock inputs or anything!  I haven't actually _defined_ any unit tests for it yet, though.

Reply:
I think the unit test is probably a good idea for future maintenance.